src/templates/blog-post.module.css

```diff
@@ -23,9 +23,10 @@
   max-width: 48rem;
 }
 
-.article h2:first-of-type {
+/*.article h2:first-of-type {
   margin-top: 0;
 }
+*/
```

Comment:
If we don't need this anymore, let's just remove it instead of commenting it out. If we need it again in the future, it will be in git history.

Reply:
Okay, I will delete